qt/widgets/common/src/PropertyHandler.cpp

```diff
@@ -1079,23 +1080,49 @@ void PropertyHandler::fix(const QString &parName) {
 }
 
 /**
- * Remove the tie.
- * @param prop :: The tie property to remove
- */
+* Remove the tie.
+* @param prop :: The tie property to remove
+* @param globalName :: Name of the parameter in compoite function
+* (e.g. f1.omega)
+*/
+void PropertyHandler::removeTie(QtProperty *prop, std::string globalName) {
+	QString parName = m_ties.key(prop, "");
```

Comment:
The indentation level seems to jump around from 2 to 4 to 8 in these changes, along with the brace positioning. Not sure why the clang-format check hasn't caught this?

Reply:
@martyngigg Do you know why the clang-format checks aren't catching this?

Reply:
@edwardb96 Nice catch. It looks like the clang-format job was missing looking at the `qt/widgets` directory so we've got a whole bunch of files that are not formatted correctly in there now.

Reply:
Ok, thanks